pkgs/servers/ankisyncd/default.nix

```diff
@@ -60,7 +60,7 @@ python3.pkgs.buildPythonApplication rec {
   meta = with lib; {
     description = "Self-hosted Anki sync server";
     maintainers = with maintainers; [ matt-snider ];
-    homepage = "https://github.com/tsudoko/anki-sync-server";
+    homepage = "https://github.com/ankicommunity/anki-sync-server";
     license = licenses.agpl3;
```

Comment:
agpl3Plus or agpl3Only?

Reply:
Hey @SuperSandro2000 thanks for taking a look. I didn't realize `licenses.agpl3` was deprecated. I had a look at [the license](https://github.com/ankicommunity/anki-sync-server/blob/develop/COPYING) as well as the licenses of other packages with `agpl3Plus` and `agpl3Only`. It isn't clear to me how I can differentiate between them. Any help would be appreciated!

Reply:
If they somewhere mention that you can use aGPL3 or at your option any later version it is Plus. I can't find anything like that so it is only.

Reply:
Okay, I updated it to agpl3Only - thanks!